README.md

```diff
@@ -1,2 +0,0 @@
-# python-test
```

Comment:
the instruction should be in this file, you should remove the one that you made in homework-1 folder

Reply:
so when I go to your repo I can see the instructions right away:

![image](https://user-images.githubusercontent.com/13141047/158013637-e7503ee8-b009-44bf-ae6d-03a6c037c4e3.png)